packages/adp-tooling/templates/project/package.json

```diff
@@ -15,7 +15,10 @@
         "@sap-ux/ui5-proxy-middleware": "^1.3.0",
         "@sap-ux/deploy-tooling": "^0.11.7"<%}%>,
         "@ui5/task-adaptation": "^1.3.0",
-        "@ui5/cli": "^3.9.2"
+        "@ui5/cli": "^3.9.2"<%if (locals.options?.enableTypeScript) {%>,
+        "@sapui5/types": "~1.131.1",
```

Comment:
`"@sapui5/types": "~1.131.1",` should probably consider setting the version based on the ui5 version of the application.

Reply:
@devinea Would using `getTypesVersion` from https://github.com/SAP/open-ux-tools/blob/b6957f47fe928e821fb1b20cf33d4472efbb2ce6/packages/ui5-config/src/utils.ts#L49 be sufficient?

Reply:
Here is the change: [113fe62](https://github.com/SAP/open-ux-tools/pull/2812/commits/113fe62e8ef2e370b7bfb8a52e60bcf85fa55638)